test/src/androidTest/kotlin/io/realm/shared/notifications/RealmListNotificationsTests.kt

```diff
@@ -33,38 +47,208 @@ class RealmListNotificationsTests : NotificationTests {
     }
 
     @Test
-    @Ignore
     override fun initialElement() {
-        TODO("Waiting for RealmList support")
+        val dataSet = OBJECT_VALUES
+
+        realm.writeBlocking {
+            val managedContainer = copyToRealm(RealmListContainer())
+            val managedDataSet = dataSet.map { copyToRealm(it) }
+            managedContainer.objectListField
+                .addAll(managedDataSet)
```

Comment:
Couldn't you just add the unmanaged set?

Reply:
We can't add unmanaged objects to a managed list as per the restriction we have in `RealmInterop.to_realm_value`.

Reply:
But shouldn't we just import them like we do when assigning fields in https://github.com/realm/realm-kotlin/blob/master/packages/library/src/commonMain/kotlin/io/realm/internal/RealmObjectHelper.kt#L107